lib/commons/aria/is-implicit-label.js

```diff
@@ -0,0 +1,24 @@
+/* global aria, dom, text */
+
+/**
+ * Note:
+ * This method is well tested via the check `implicit-label`
+ * See - `test/checks/label/implicit.js`
+ */
+
+/**
+ * Check if given element has `implicit` label
+ * @method isImplicitLabel
+ * @memberof axe.commons.aria
+ * @param {Object} virtualNode virtualNode
+ * @return {Boolean}
+ */
+aria.isImplicitLabel = function(virtualNode) {
```

Comment:
I don't think you should pull this into a commons. See my comment about using `text.textLabel`.